taxi/forms.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from django.forms import CheckboxSelectMultiple
+
+from .models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number", )
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),

```

Comment:
The `queryset` for the `drivers` field in `CarForm` should be limited to `Driver` instances. Currently, it uses `get_user_model().objects.all()`, which includes all user types. Consider using `Driver.objects.all()` to ensure only drivers are selectable.